lib/include/prjxray/xilinx/bitstream_reader.h

```diff
@@ -77,10 +85,61 @@ class BitstreamReader {
 
        private:
 	static std::array<uint8_t, 4> kSyncWord;
+	const std::vector<uint32_t> kWcfgCmd = {0x30008001, 0x1};
```

Comment:
Comment on where these magic numbers come from?  Also, please use `constexpr std::array` rather than `std::vector`.

Reply:
Changed to `static const std::array`. `constexpr` requires to be `static constexpr` to pass compilation. However, I get linker errors in methods which use these variables. I briefly found this [explanation] (https://stackoverflow.com/questions/43193749/linker-error-for-constexpr-static-member-variable-in-gcc-and-clang?rq=1)